tests/deprecated_classes.py

```diff
@@ -15,6 +15,10 @@
 # specific language governing permissions and limitations
 # under the License.
 HOOKS = [
+    (
```

Comment:
This import is not valid. We do not expect users to import BaseHook by:
```
from airflow.hooks.filesystem import BaseHook
```
BaseHook should be imported at all times, as shown below.
```
from airflow.hooks.base_hook import BaseHook
```
There is a broken backwards compatibility here, but we can't prevent API misuse as Python is too flexible